lib/workers/repository/onboarding/branch/index.spec.ts

```diff
@@ -13,6 +14,7 @@ import {
 } from '../../../../constants/error-messages';
 import { Pr } from '../../../../platform';
 import { PrState } from '../../../../types';
+import * as cache_ from '../../../../util/cache/repository';
```

Comment:
```suggestion
import * as _cache from '../../../../util/cache/repository';
```
All other import are using a prefix underscore, so unify